### PR TITLE
update npm deps and fix issue with sharp 0.31.2 in alpine lts

### PR DESCRIPTION
**Pull request by infokiller**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "Firenvim",
-  "version": "0.2.13",
+  "version": "0.2.15",
```
[the remark this answers is not in the document]
Comment (infokiller) [reply package-lock.json]: Thanks you for the awesome project and quick response!

**Comment by infokiller** [comment]
Hey @glacambre great to talk to you again and thanks for the quick response!
Sorry for the failing CI and the merge noise, I will take a look later today (will try your patch of course- thanks a lot!).

**Comment by infokiller** [comment]
@glacambre thanks, indeed I didn't run `npm install` after `npm update`, which caused the issue.
Now most of the CI tests pass, but two failed with "artifact expired". Is this benign?

**Comment by glacambre** [comment]
@infokiller Thanks! Yes, these failures can be ignored. The first one failed in the code coverage comparison step (it tries to compare the current run's coverage against the last run on master, which happened too long ago), the second one failed due to a flaky test :).

**Pull request merged**